### DISCUSSION TITLE
Twitter Rtwit problem

### DISCUSSION THREAD

**Discussion by abdurrahman65** [IBAX-io/go-ibax#1713] [Soru Cevap - Türkçe]
Twitter Rtwit sorununu yoğunluktan dolayı olmuyor olmalı bazıları yapmış düzelir şimdi

**Comment by Zeynel26477575** [comment]
Bende de aynı problem var.

**Comment by ibax2525** [comment]
problem devam ediyor

**Comment by Yusufknk** [comment]
Retweet görevleri için belirli sayıda API kontenjanı var bu kontenjan son retweet görevlerinde az sayıda kaldığı için ilk yapanlar doldurdu. Alınan hataların ve çoğu kişinin yapamamasinin nedeni bu
